src/less/less/ds6/variables.less

```diff
@@ -317,7 +317,80 @@
 @ds6-icon-arrow-right-base64: 'PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyMSIgaGVpZ2h0PSIyMCI+PHBhdGggZD0iTTE2LjMxNyAxMC45NzZzLTYuNDEzIDYuMTI5LTYuOSA2LjU5N2MtLjQ5LjQ3LS40MzIgMS4yMzYgMCAxLjY1LjQzLjQxMiAxLjI2Ny40NDMgMS43MjMgMEMxMS41OTggMTguNzggMjEgOS43OTggMjEgOS43OThTMTEuNzA4Ljk0MyAxMS4xNC4zNzNjLS41NjYtLjU2OC0xLjI4My0uNDIxLTEuNzIzIDAtLjQ0Mi40MjItLjU2IDEuMDkzIDAgMS42NS41NTYuNTU2IDYuOSA2LjU5NyA2LjkgNi41OTdIMS4yODJDLjQxNCA4LjYyIDAgOS4yMDggMCA5Ljc5OHMuNDEgMS4xNzggMS4yODIgMS4xNzhoMTUuMDM1eiIvPjwvc3ZnPg==';
 @ds6-icon-arrow-right-disabled-base64: 'PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyMSIgaGVpZ2h0PSIyMCIgZmlsbD0iI2M3YzdjNyI+PHBhdGggZD0iTTE2LjMxNyAxMC45NzZzLTYuNDEzIDYuMTI5LTYuOSA2LjU5N2MtLjQ5LjQ3LS40MzIgMS4yMzYgMCAxLjY1LjQzLjQxMiAxLjI2Ny40NDMgMS43MjMgMEMxMS41OTggMTguNzggMjEgOS43OTggMjEgOS43OThTMTEuNzA4Ljk0MyAxMS4xNC4zNzNjLS41NjYtLjU2OC0xLjI4My0uNDIxLTEuNzIzIDAtLjQ0Mi40MjItLjU2IDEuMDkzIDAgMS42NS41NTYuNTU2IDYuOSA2LjU5NyA2LjkgNi41OTdIMS4yODJDLjQxNCA4LjYyIDAgOS4yMDggMCA5Ljc5OHMuNDEgMS4xNzggMS4yODIgMS4xNzhoMTUuMDM1eiIvPjwvc3ZnPg==';
 @ds6-icon-arrow-right-focus-base64: 'PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyMSIgaGVpZ2h0PSIyMCIgZmlsbD0iIzAwNmVmYyI+PHBhdGggZD0iTTE2LjMxNyAxMC45NzZzLTYuNDEzIDYuMTI5LTYuOSA2LjU5N2MtLjQ5LjQ3LS40MzIgMS4yMzYgMCAxLjY1LjQzLjQxMiAxLjI2Ny40NDMgMS43MjMgMEMxMS41OTggMTguNzggMjEgOS43OTggMjEgOS43OThTMTEuNzA4Ljk0MyAxMS4xNC4zNzNjLS41NjYtLjU2OC0xLjI4My0uNDIxLTEuNzIzIDAtLjQ0Mi40MjItLjU2IDEuMDkzIDAgMS42NS41NTYuNTU2IDYuOSA2LjU5NyA2LjkgNi41OTdIMS4yODJDLjQxNCA4LjYyIDAgOS4yMDggMCA5Ljc5OHMuNDEgMS4xNzggMS4yODIgMS4xNzhoMTUuMDM1eiIvPjwvc3ZnPg==';
-
+@ds6-icon-arrow-move-base64:
+'PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB3aWR0aD0iMTIiIGhlaWdodD0iMTIiPjxkZWZzPjxwYXRoIGlkPSJhIiBkPSJNMy4yOTcgMS45MjRoNy42OWMuNTQ2LS4wMDEuOTYyLS40ODEuOTYyLS45NjFzLS40NDgtLjk2Ni0uOTYxLS45NjNDMTAuNDc1LjAwNCAwIDAgMCAwcy4wMTcgMTAuMzQgMCAxMC45ODljLS4wMTUuNjQ3LjQ3Ljk2MS45NjMuOTYxLjQ5IDAgLjk0Ny0uMzI1Ljk2MS0uOTYxLjAxNC0uNjM1IDAtNy42OTIgMC03LjY5Mmw4LjM3NyA4LjM3OWMuNDgzLjQ4MiAxLjA1OC4zNyAxLjQwMS4wMjcuMzQ0LS4zNDMuNDU4LS45MTYtLjAyNy0xLjQwMUwzLjI5NyAxLjkyNHoiLz48L2RlZnM+PHVzZSBmaWxsPSIjMDAwIiB4bGluazpocmVmPSIjYSIgZmlsbC1ydWxlPSJldmVub2RkIi8+PC9zdmc+';
```

Comment:
Why are there extra new lines before these new strings?

Reply:
Oh, that's weird. Didn't show up in my editor. Will fix those.